java.jdbc-ballerina/Package.md

```diff
@@ -25,7 +25,7 @@ groupId = "com.h2database"
 
 ### Client
 To access a database, you must first create a 
-[jdbc:Client](https://ballerina.io/swan-lake/learn/api-docs/ballerina/#/jdbc/clients/Client) object. 
+[jdbc:Client](https://ballerina.io/swan-lake/learn/api-docs/ballerina/#/java.jdbc/clients/Client) object. 
```

Comment:
If we are moving out of the distribution, docs will not be created, will it?

Reply:
These docs will create and appear in Ballerina central.

Reply:
Ack